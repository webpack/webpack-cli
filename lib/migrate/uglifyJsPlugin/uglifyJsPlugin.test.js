"use strict";

const defineTest = require("../../utils/defineTest");

defineTest(__dirname, "uglifyJsPlugin", "uglifyJsPlugin-0");
defineTest(__dirname, "uglifyJsPlugin", "uglifyJsPlugin-1");
defineTest(__dirname, "uglifyJsPlugin", "uglifyJsPlugin-2");
defineTest(__dirname, "uglifyJsPlugin", "uglifyJsPlugin-3");
defineTest(__dirname, "uglifyJsPlugin", "uglifyJsPlugin-4");
