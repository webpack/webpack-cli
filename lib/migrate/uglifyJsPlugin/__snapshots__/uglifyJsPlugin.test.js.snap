// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uglifyJsPlugin transforms correctly using "uglifyJsPlugin-0" data 1`] = `
"module.exports = {
    plugins: [],

    optimization: {
        minimize: true
    }
}
"
`;

exports[`uglifyJsPlugin transforms correctly using "uglifyJsPlugin-1" data 1`] = `
"const UglifyJsPlugin  = require('uglifyjs-webpack-plugin');
module.exports = {
    devtool: \\"source-map\\",
    plugins: [],

    optimization: {
        minimizer: [new UglifyJsPlugin({
            sourceMap: true,
            compress: {}
        })]
    }
}
"
`;

exports[`uglifyJsPlugin transforms correctly using "uglifyJsPlugin-2" data 1`] = `
"const Uglify  = require('uglifyjs-webpack-plugin');
module.exports = {
    devtool: \\"source-map\\",
    plugins: [],

    optimization: {
        minimizer: [new Uglify({
            sourceMap: true,
            compress: {}
        })]
    }
}
"
`;
